pkg/context/project.go

```diff
@@ -35,19 +41,20 @@ func ResolveProject(f *pflag.FlagSet) {
 		}
 		common.ProjectId = resolvedProject.Projects[0].Id
 	} else {
-		activeProject := common.GetActiveProject()
+		activeProject := user_project.GetActiveProject()
 		resolvedProject := project.GetProject(activeProject)
 		if len(resolvedProject.Projects) == 0 {
 			initActiveProject()
 			common.CliExit(errors.New(fmt.Sprintf("Active project '%v' does not exist, or you do not have access "+
-				"to it. The following project has been set instead: %v", activeProject, common.GetActiveProject())))
+				"to it. The following project has been set instead: %v", activeProject, user_project.GetActiveProject())))
 		}
 		common.ProjectId = resolvedProject.Projects[0].Id
 	}
 }
 
 func SetActiveProject(projectName string) {
 	if len(project.GetProject(projectName).Projects) != 0 {
+		user_project.LoadActiveProject()
```

Comment:
Is this needed if you save a new active project 1 line later?

Reply:
because we need to load the whole file (which can have active project from other users) so that the data of other users is not lost. When it's loaded the active project of the currently logged in users will be changed.